cmd/dep/root_analyzer.go

```diff
@@ -0,0 +1,182 @@
+// Copyright 2016 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"encoding/hex"
+
+	"github.com/golang/dep"
+	fb "github.com/golang/dep/internal/feedback"
+	"github.com/golang/dep/internal/gps"
+)
+
+// importer handles importing configuration from other dependency managers into our format
+type importer interface {
+	Name() string
+	Import(path string, pr gps.ProjectRoot) (*dep.Manifest, *dep.Lock, error)
```

Comment:
Seems like `importer` should just compose `gps.ProjectAnalyzer`, rather than having a new method called `Import` with the same (well, an equivalent) signature, and the same basic purpose, as `DeriveManifestAndLock`. Is there a reason not to do that?